toronto/src/pages/Controversy.jsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+import { ControversyVote } from '@/components/molecules';
+import { Header, Text } from '@/components/atoms';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useCallback, useEffect, useState } from 'react';
+import { getPostApi, postCommentApi } from '../api/Api';
+
+const Wrapper = styled.div`
+  height: 100vh;
+  padding: 150px 320px;
+  display: flex;
+  flex-direction: column;
+  background-color: #f9fafb;
+`;
+
+const Controversy = () => {
+  const [data, setData] = useState({});
+  const { postId } = useParams();
+  const navigate = useNavigate();
+
+  const getpostData = useCallback(async () => {
+    const postData = await getPostApi(postId);
+    const postTitleData = JSON.parse(postData.data.title);
+    setData({
+      postTitle: postTitleData.postTitle,
+      postContent: postTitleData.postContent,
+      agreeContent: postTitleData.agreeContent,
+      disagreeContent: postTitleData.disagreeContent,
+      image: postData.data.image,
+    });
+  }, [postId]);
+
+  useEffect(() => {
+    getpostData();
+  }, [getpostData]);
+
+  const handleChange = async (opinionState) => {
+    if (!opinionState || !postId) return;
+    await postCommentApi({
+      comment: JSON.stringify({
+        type: 'vote',
+        content: opinionState,
+      }),
+      postId: postId,
+    });
+    navigate(`/controversy/result/${postId}`);
+  };
+  return (
+    <div>
+      {data && (
+        <Wrapper>
+          <Header>{data.postTitle}</Header>
+          <Text style={{ marginBottom: '24px' }}>{data.postContent}</Text>
+          <ControversyVote
+            agreeTitle={data.agreeContent}
+            disagreeTitle={data.disagreeContent}
+            imgSrc={data.image}
+            onChange={handleChange}
```

Comment:
`React.children` 관련해서 생기는 에러는 아래처럼 수정하면 해결이 가능하긴 합니다! 하지만 좀 더 근본적인 해결책이 있어야 하긴 하겠네요!

```suggestion
          <Header>{data.postTitle || ''}</Header>
          <Text style={{ marginBottom: '24px' }}>{data.postContent || ''}</Text>
          <ControversyVote
            agreeTitle={data.agreeContent || ''}
            disagreeTitle={data.disagreeContent || ''}
            imgSrc={data.image || ''}
            onChange={handleChange}
```

Reply:
넵 그럼 이부분은 추후 이야기를 해보죠 :)